L/libqasm/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libqasm"
+version = v"0.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/QuTech-Delft/libqasm.git", "1f317077e2f2462cafec8385666d6e996340d2e8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mkdir -p libqasm/build
+cd libqasm/build/
+# rm /usr/share/cmake/Modules/Compiler/._*
+# rm /usr/share/cmake/Modules/CompilerId/._*
+# rm /usr/share/cmake/Modules/._*
+git submodule init
+git submodule update
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DLIBQASM_COMPAT=ON -DBUILD_SHARED_LIBS=ON -DLIBQASM_BUILD_PYTHON=OFF ..
+export PATH=$PATH:/workspace/srcdir/libqasm/build/src/cqasm/tree-gen/:/workspace/srcdir/libqasm/build/src/cqasm/func-gen/
+make -j
```

Comment:
Please specify `-j{nproc}`, `${nproc}` is set to a value which doesn't oversubscribe the machine.